Source/RunActivity/Viewer3D/Popups/HelpWindow.cs

```diff
@@ -1152,6 +1183,231 @@ private void writeline()
+
+            foreach (var car in playerTrain.Cars)
+            {
+                // ignore (legacy) EOT
+                if (car.WagonType == TrainCar.WagonTypes.EOT || car.CarLengthM < 1.1f) { continue; }
+
+                var wag = car is MSTSWagon ? (MSTSWagon)car : null;
+                var eng = car is MSTSLocomotive ? (MSTSLocomotive)car : null;
+
+                var isEng = (car.WagonType == TrainCar.WagonTypes.Engine && eng != null && eng.MaxForceN > 25000);  // count legacy driving trailers as wagons
+
+                if (car.IsMetric) { isMetric = true; }; if (car.IsUK) { isUK = true; }
+
+                if (isEng)
+                {
+                    engCount++;
+                    numAxles += eng.LocoNumDrvAxles + eng.GetWagonNumAxles();
+                    totPowerW += eng.MaxPowerW;
+                    totMaxTractiveEffortN += eng.MaxForceN;
+                    engMaxTractiveForceN += eng.MaxForceN;
+                    totMaxDynamicBrakeForceN += eng.MaxDynamicBrakeForceN;
+                    engMaxDynBrakeForceN += eng.MaxDynamicBrakeForceN;
+
+                    // hanlde transition from wagons to engines
+                    if (wagMassKg > 0)
+                    {
+                        sectionMass += massSeparator + FormatStrings.FormatLargeMass(wagMassKg, isMetric, isUK);
+                        wagMassKg = 0f; massSeparator = "  +  ";
+                    }
+                }
+                else if (wag != null)
+                {
+                    numCars++;
+                    numAxles += wag.GetWagonNumAxles();
+                    wagMassKg += wag.MassKG;
+
+                    // handle transition from engines to wagons
+                    if (engCount > 0 || (engCount == 0 && numCars == 0))
+                    {
+                        numEngines += countSeparator + engCount.ToString();
+                        engCount = 0; countSeparator = "+";
+                        sectionMaxTractiveForce += forceSeparator + FormatStrings.FormatForce(engMaxTractiveForceN, isMetric);
+                        sectionMaxDynamicBrakeForce += forceSeparator + FormatStrings.FormatForce(engMaxDynBrakeForceN, isMetric);
+                        engMaxTractiveForceN = 0f; engMaxDynBrakeForceN = 0f;  forceSeparator = "  +  ";
+                    }
+                }
+
+                // wag and eng
+                if (wag != null)
+                {
+                    maxBrakeForceN += wag.MaxBrakeForceN;
+                    var couplerStrength = GetMinCouplerStrenght(wag);
+                    if (couplerStrength < lowestCouplerStrengthN) { lowestCouplerStrengthN = couplerStrength; }
+                    var derailForce = GetDerailForce(wag);
+                    if (derailForce < lowestDerailForceN) { lowestDerailForceN = derailForce; }
+                    if (wag.MaxBrakeForceN > 0) { numOperativeBrakes++; }  // simplistic: real world has cutout brakes, and no brakes on some articulated car-segments
```

Comment:
A better check may be to consider the wagon's brake equipment. If the wagon has any brake valve, count it as an operative brake.

```suggestion
                    if (wag.BrakeValve != MSTSWagon.BrakeValveType.None) { numOperativeBrakes++; }  // simplistic: real world has cutout brakes, and no brakes on some articulated car-segments
```

That's still not foolproof (some content creators will still put a triple valve on every section of an articulated unit), but closer to the idea of an operative brake.

Using brake force alone could mistakenly count something with handbrakes only as an operative brake, when that should not count according to the rulebooks.

Reply:
This is a bit more involved. 

For the Content Manager, I would have to replicate the following code from MSTSWagon.cs:

```
                case "wagon(brakeequipmenttype":
                    foreach (var equipment in stf.ReadStringBlock("").ToLower().Replace(" ", "").Split(','))
                    {
                        switch (equipment)
                        {
                            case "triple_valve": BrakeValve = BrakeValveType.TripleValve; break;
                            case "distributor":
                            case "graduated_release_triple_valve": BrakeValve = BrakeValveType.Distributor; break;
                            case "distributing_valve": BrakeValve = BrakeValveType.DistributingValve; break;
                            case "emergency_brake_reservoir": EmergencyReservoirPresent = true; break;
                            case "handbrake": HandBrakePresent = true; break;
                            case "auxilary_reservoir": // MSTS legacy parameter - use is discouraged
                            case "auxiliary_reservoir":
                                AuxiliaryReservoirPresent = true;
                                break;
                            case "manual_brake": ManualBrakePresent = true; break;
                            case "retainer_3_position": RetainerPositions = 3; break;
                            case "retainer_4_position": RetainerPositions = 4; break;
                            case "supply_reservoir":
                                SupplyReservoirPresent = true;
                                break;
                        }
                    }
                    break;
```
Also, this code does not seem to cover all brake equipment. MSTS has cars with "EP_brake" and "Vacuum_brake" in the list. I am not sure how relevant these are.

Second, I found some modern equipment that has wrong values.
* The Amtrak cars in the BNSF Starter Route are missing BrakeEquipmentType, but have MaxBrakeForce . In the same package, EOT devices have Triple_valve in the equipment, but a MaxBrakeForce  of 0 (zero).

Regarding the hand brakes, there is a separate config, MaxHandbrakeForce. 

So, the question is what is the least wrong looking forward.

As to making the change for the Train Info window and Content Manager:

* I am fine to use `BrakeValveType.None` in the F1 Train Info window.
* I am reluctant to copy the above code into the Content Manager.
* I could refactor the above code into a method, and call the method from the Content Manager. I don't know if or how much that will affect the linking of the content manager.
* I could make an assumption that anything other than `Handbrake` probably means that the car has brakes. But that will not match the TrainInfo.

Reply:
So the EP_brake and Vacuum_brake equipment types led me down a bit of a rabbit hole. Those equipment types, like any equipment type not on the list, don't actually do anything. This means that those trains have a BrakeValve of none, but work anyway. Didn't know that was an allowed state, but this means the BrakeValve isn't foolproof.

The next best thing would be to check the brake system type. If the brake system is SingleTransferPipe or ManualBraking, you know for sure that it won't be simulated with any sort of automatic brake valve. The other brake system types can count as operative. That could be done like this:
```suggestion
                    if (car.BrakeSystem != null && !((car.BrakeSystem is SingleTransferPipe) || (car.BrakeSystem is ManualBraking)) && wag.MaxBrakeForceN > 0) { numOperativeBrakes++; }  // simplistic: real world has cutout brakes, and no brakes on some articulated car-segments
```

Not sure what you mean by starter route wagons missing BrakeEquipmentType, I don't see anything missing there. Besides, if BrakeEquipmentType was missing the brakes wouldn't work at all, which you'd probably want to count as a non-functional brake valve. Similar story if BrakeSystemType was missing, then it gets set as SingleTransferPipe, which shouldn't count as functional either.

For the content manager, the brake system type might be easier, as you just need to read the one string in 'BrakeSystemType' and if it's (converted to lowercase) "manual_braking", "air_piped", or "vacuum_piped" then it definitely won't have any automatic brakes.

Reply:
> Not sure what you mean by starter route wagons missing BrakeEquipmentType, I don't see anything missing there. 

Example: `TS_STARTER_ROUTE.zip\TRAINS\trainset\SLI_AMT_EMPIRE_BUILDER\SUPERLINER1_P4_COACH_34001.wag`, 
 extracted from `https://ts-files.com/files/TS_STARTER_ROUTE.zip`. It matches what my ORTS instance has installed (via the content form). Here is the relevant part:
```
	BrakeSystemType( "Air_single_pipe" )
	MaxBrakeForce( 24.3kN )

	MaxHandbrakeForce( 57.2kN )
	NumberOfHandbrakeLeverSteps( 100 )

	EmergencyBrakeResMaxPressure( 90 )
	TripleValveRatio( 2.5 )
	MaxReleaseRate( 50 )
	MaxApplicationRate( 50 )
	MaxAuxilaryChargingRate( 1.684 )
	EmergencyResCapacity( 2.604 )
	EmergencyResChargingRate( 1.684 )
	BrakeCylinderPressureForMaxBrakeBrakeForce( 77 )
```
There is no `BrakeEquipmentType` (and there is no include).

I like your suggestion on using `BrakeSystemType`. I will go with that,